### PR TITLE
Update to version 0.22.187631

### PR DESCRIPTION
**Pull request by msoeken**

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-
----------------------------------------------------------
-
-FluentAssertions 5.10.3 - Apache-2.0
-
-
-(c) 2008 VeriSign, Inc.
-Copyright Dennis Doomen
-
+NOTICES AND INFORMATION
```
Comment (tcNickolas) [inline NOTICE.txt]: Should this file be part of the change? I couldn't find any difference in what GitHub highlights as diff

**Review by tcNickolas — APPROVED** [review]
Looks good, just one file I'm not sure about

File: NOTICE.txt (hunk 1 above)
Comment (msoeken) [reply NOTICE.txt]: I think there is an error in Github. The change is just in one line for the license of Q# compiler.

**Pull request merged**